### DISCUSSION TITLE
ttk.Meter

### DISCUSSION THREAD

**Discussion by Ginias2000** [israel-dryer/ttkbootstrap#599] [Q&A]
how can i change the Meter bg_color by defaut it taking the root style bg but i want its bg to be diferent i used every method online but it not working

**Comment by TheDemonGuard** [comment]
@Ginias2000 
https://github.com/israel-dryer/ttkbootstrap/discussions/763#discussioncomment-15054953

> how can i change the Meter bg_color by defaut it taking the root style bg but i want its bg to be diferent i used every method online but it not working

**Reply by israel-dryer** [reply]
Currently, there is no means of changing the inner circle background color. The indicator is an image with a transparent background so that the label part shines through. This may be able to be augmented in the future.
